elements/pfe-sass/mixins/_components.scss

```
@@ -246,3 +246,84 @@
     }
   }
 }
+
+/// ===========================================================================
+/// List styles
+/// ===========================================================================
+
+// Mixins and extends can't use the LOCAL-VARIABLES map because we would be
+// at risk of overriding an elements map if they call our mixin. We need to
+// use just regular sass variables and pfe-local and specify the $fallbacks.
+@mixin ordered-list-primary($size: 56px) {
+  // First we create our Sass variables that we will use for fallbacks
+  // The gap between list items
+  $gap: pfe-var(content-spacer);
+  // The space between the circle and content
+  $spacer: pfe-var(content-spacer);
+  // rgb value of the color theme of the circle
+  // we need to wrap this in a calc function so it can be dynamic.
+  $color: (238, 0, 0);
+  $font-size: $pf-global--FontSize--2xl;
+
+  list-style: none;
+  min-height: pfe-local(gap, $fallback: $gap);
+  margin: pfe-local(gap, $fallback: $gap);
+  padding-left: pfe-local(PaddingLeft, $fallback: 0);
+
+  & > li {
+    // increment the local counter variable
+    counter-increment: pfe-list;
+    position: relative;
+    min-height: pfe-local(size, $region: circle, $fallback: $size);
+    // the li content will start after the circle and a defined spacer
+    padding-left: calc(#{pfe-local(spacer, $fallback: $spacer)} + #{pfe-local(size, $region: circle, $fallback: $size)});
+    // the li content needs to align with the middle of the radius but we
+    // need to account for the fact that it's also calculating the size
+    // of the line height of the number inside of the circle
+    padding-top: calc((#{pfe-local(size, $region: circle, $fallback: $size)} / 2) - ((#{pfe-local(FontSize, $region: circle, $fallback: $font-size)} / 2)));
+  }
+  & > li:before {
```

small detail to get it looking exactly like the mockups
```suggestion
  & > ul {
    list-style-type: disc;
  }
  & > li:before {
```